app/controllers/collections_controller.rb

```diff
@@ -25,11 +25,23 @@ def load_collection_from_id
 
   def index
```

Comment:
I ended up deleting the N+1 spec for this method, as it was flagging something unrelated and we want to move collections to Elasticsearch anyways (also I'd like to keep the PR from getting too much out of hand :see_no_evil:)